spec/integration/customer_spec.rb

```diff
@@ -16,18 +15,22 @@
 
   let(:expected_collection_length) { 1 }
 
-  specify { client.Customer.should respond_to(:find_by_email) }
-  specify { client.Customer.should respond_to(:find_by_name) }
+  it :find_by_email do
```

Comment:
Just a small style thing.  We would generally use `specify` here rather than `it`, and in all the other cases where `it` doesn't produce a sensible sentence.  So for example below: `it "returns a collection of customers from a search" ...` makes sense whereas `it :find_by_email ...` doesn't.